app/routes/report-routes.ts

```diff
@@ -172,7 +172,9 @@ router.post('/', createReportRateLimit, async (req, res) => {
 
   reportRepo.addNewCovidReport(passcode, covidReport);
   if (req.body['passcode']) {
-    return res.redirect(`${res.locals.urls.profile}/${passcode}?success=true`);
+    return res.redirect(
+      `${res.locals.urls.profile}/${passcode}?success=true#contribute`
```

Comment:
Did you test this? Should not #contribute come before the query param?

Reply:
yeah, head some recent experience with it. also tested it ;)